src/app/api/routes/users.py

```diff
@@ -18,27 +20,31 @@ async def get_my_user(me: UserRead = Security(get_current_user, scopes=["me"])):
 
 @router.put("/update-info", response_model=UserRead)
 async def update_my_info(payload: UserInfo, me: UserRead = Security(get_current_user, scopes=["me"])):
-    return await routing.update_entry(users, payload, me.id)
+    return await crud.update_entry(users, payload, me.id)
 
 
 @router.put("/update-pwd", response_model=UserInfo)
 async def update_my_password(payload: Cred, me: UserRead = Security(get_current_user, scopes=["me"])):
-    return await routing.update_pwd(users, payload, me.id)
+    entry = await crud.get_entry(users, me.id)
+    await update_access_pwd(payload, entry["access_id"])
```

Comment:
same comments as in the device route. I would find it cleaner to have separate dedicated functions called by the route. Same for create_user and upate_user_password